crypto/keys/lazy_keybase_keyring.go

```diff
@@ -0,0 +1,323 @@
+package keys
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/99designs/keyring"
+	"github.com/cosmos/cosmos-sdk/client/input"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/crypto/bcrypt"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+var _ Keybase = lazyKeybaseKeyring{}
+
+type lazyKeybaseKeyring struct {
```

Comment:
what's the purpose of `lazyKeybaseKeyring` (add comment)

Reply:
not sure, it was set up previously by someone else